Nodes/Scenes/Menu/TitleScreen.tscn

```diff
@@ -33,98 +53,109 @@ __meta__ = {
 [node name="MarginContainer" type="MarginContainer" parent="."]
 anchor_top = 0.5
 anchor_bottom = 0.5
-margin_top = -72.0
-margin_right = 250.0
-margin_bottom = 72.0
+margin_top = -185.0
+margin_right = 1094.0
+margin_bottom = 185.0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Title" type="Label" parent="MarginContainer"]
-visible = false
-margin_right = 1183.0
-margin_bottom = 339.0
-grow_horizontal = 2
-grow_vertical = 2
-rect_min_size = Vector2( 10, 10 )
-size_flags_vertical = 5
-custom_colors/font_color = Color( 0.160784, 1, 0.980392, 1 )
-text = "BINDING TOWER"
-align = 1
-valign = 1
-
 [node name="HBoxContainer" type="HBoxContainer" parent="MarginContainer"]
-margin_right = 1183.0
-margin_bottom = 339.0
+margin_right = 1094.0
+margin_bottom = 370.0
 custom_constants/separation = 400
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="Menu" type="VBoxContainer" parent="MarginContainer/HBoxContainer"]
-margin_right = 250.0
-margin_bottom = 339.0
+margin_right = 244.0
+margin_bottom = 370.0
 custom_constants/separation = 40
 alignment = 1
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Logo" type="TextEdit" parent="MarginContainer/HBoxContainer/Menu"]
-visible = false
-margin_right = 12.0
-text = "Binding Tower
-LMAO
-BINDING TOWER"
-
-[node name="TextureButton" type="TextureButton" parent="MarginContainer/HBoxContainer/Menu"]
-visible = false
-margin_right = 250.0
-
 [node name="btn_continue" type="Button" parent="MarginContainer/HBoxContainer/Menu"]
-margin_top = 31.0
-margin_right = 250.0
-margin_bottom = 73.0
-icon = ExtResource( 1 )
+margin_top = 17.0
+margin_right = 244.0
+margin_bottom = 71.0
+size_flags_horizontal = 0
+custom_fonts/font = ExtResource( 2 )
+text = "Continue"
 flat = true
+script = ExtResource( 7 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="btn_newGame" type="Button" parent="MarginContainer/HBoxContainer/Menu"]
-margin_top = 113.0
-margin_right = 250.0
-margin_bottom = 155.0
-icon = ExtResource( 2 )
+margin_top = 111.0
+margin_right = 244.0
+margin_bottom = 165.0
+size_flags_horizontal = 0
+custom_fonts/font = ExtResource( 2 )
+text = "New Game"
 flat = true
+script = ExtResource( 7 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="btn_options" type="Button" parent="MarginContainer/HBoxContainer/Menu"]
+<<<<<<< HEAD
+margin_top = 205.0
+margin_right = 212.0
+margin_bottom = 259.0
+size_flags_horizontal = 0
+custom_fonts/font = ExtResource( 2 )
+text = "Options"
+=======
 margin_top = 195.0
 margin_right = 250.0
 margin_bottom = 247.0
 icon = ExtResource( 4 )
+>>>>>>> origin
```

Comment:
I think this is a remnant of a merge conflict that wasn't resolved. I'm surprised git didn't pick up on this.

Reply:
Please look at the updated file.